tests/fdb/test_fdb_flush.py

```diff
@@ -0,0 +1,362 @@
+
+    @pytest.fixture(scope="class", autouse=True)
+    def validateDummyMacAbsent(self, duthosts, rand_one_dut_hostname):
+        """
+            Validates that test/dummy MAC entry is absent before the test runs
+
+            Args:
+                duthost (AnsibleHost): Device Under Test (DUT)
+
+            Returns:
+                None
+        """
+        duthost = duthosts[rand_one_dut_hostname]
+        pytest_assert(self.__getFdbTableCount(duthost, self.DUMMY_MAC_PREFIX) == 0, "Test dummy MAC is already present")
+
+    @pytest.fixture(scope="class", autouse=True)
+    def prepareDut(self, request, duthosts, rand_one_dut_hostname):
+        """
+            Prepare DUT for FDB test
+
+            It update the fdb_aging_time value, update the swss configuration, and restore SWSS configuration afer
+            test completes
+
+            Args:
+                request (Fixture): pytest request object
+                duthost (AnsibleHost): Device Under Test (DUT)
+
+            Returns:
+                None
+        """
+        duthost = duthosts[rand_one_dut_hostname]
+        fdbAgingTime = request.config.getoption('--fdb_aging_time')
+
+        self.__deleteTmpSwitchConfig(duthost)
+        duthost.shell(argv=["docker", "cp", "swss:/etc/swss/config.d/switch.json", "/tmp"])
+        duthost.replace(
+            dest='/tmp/switch.json',
+            regexp='"fdb_aging_time": ".*"',
+            replace='"fdb_aging_time": "{0}"'.format(fdbAgingTime),
+            backup=True
+        )
+        duthost.shell(argv=["docker", "cp", "/tmp/switch.json", "swss:/etc/swss/config.d/switch.json"])
+        self.__loadSwssConfig(duthost)
+
+        yield
+
+        result = duthost.find(path=["/tmp"], patterns=["switch.json.*"])
+        if result["matched"] > 0:
+            duthost.shell(argv=["docker", "cp", result["files"][0]["path"], "swss:/etc/swss/config.d/switch.json"])
+            self.__loadSwssConfig(duthost)
+        self.__deleteTmpSwitchConfig(duthost)
+
+    def prepare_test(self, duthosts, rand_one_dut_hostname):
+        logging.info("Start prepare_test")
+
+        # Perform FDB clean up before each test
+        fdb_cleanup(duthosts, rand_one_dut_hostname)
+   
+        duthost = duthosts[rand_one_dut_hostname]
+
+        # save existing core files list
+        self.checkDutCorefiles(duthost, True)
+        logging.info("pre_exist_cores {} ".format(self.pre_exist_cores))
+
+        # determine target test port
+        self.target_port = None
+        ifs_status = duthost.get_interfaces_status()
+        logging.info("ifs_status {} ".format(ifs_status))
+        for interface_key, interface_info in ifs_status.items():
+            if ('N\/A' != interface_info['alias']) and ('N\/A' != interface_info['type']) and ('up' == interface_info['oper']):
+                logging.debug('get interface %s' % (interface_info['interface']))
+                self.target_port = interface_info['interface']
```

Comment:
@lipxu is the target_port always the first port in the output of show interface status? Can we randomly pick up such as 3 or 5 interfaces from the list? The test coverage could be more accurate.